.github/workflows/amazon_bedrock.yml

```diff
@@ -26,7 +26,9 @@ permissions:
 env:
   PYTHONUNBUFFERED: "1"
   FORCE_COLOR: "1"
-  AWS_REGION: us-east-1
+  AWS_REGION: "us-east-1"
+  AWS_BEDROCK_GUARDRAIL_ID: ${{ secrets.AWS_BEDROCK_GUARDRAIL_ID }}
+  AWS_BEDROCK_GUARDRAIL_VERSION: "1"
```

Comment:
although not a key, since it's needed for the guardrails test to work, maybe we can also added it to the internal documentation page?

Reply:
for clarity, I have added `AWS_BEDROCK_GUARDRAIL_VERSION` to our password manager, along with `AWS_BEDROCK_GUARDRAIL_ID`